src/System.Net.WebSockets/src/System/Net/WebSockets/WebSocketException.cs

```diff
@@ -122,9 +124,15 @@ public WebSocketException(string message, Exception innerException)
         {
         }
 
+        private WebSocketException(SerializationInfo serializationInfo, StreamingContext streamingContext)
+            : base(serializationInfo, streamingContext)
+        {
```

Comment:
It makes no sense that it does not read the error code out of the blob. Why serialize it. I see desktop has the same. A bug I guess. You could fix that if you want.

Reply:
I guess to support older versions of .NET Framework that relied on that field in the blob... Haven't checked...